### PR TITLE
[11.0][MIG] account_invoice_margin

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa

### CODE REVIEW

File: account_invoice_margin/tests/test_account_invoice_margin.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+        self.invoice.invoice_line_ids.discount = 50
+        self.assertEqual(self.invoice.invoice_line_ids.margin, 0.0)
+
+    def test_invoice_margin_uom(self):
+        self.invoice.invoice_line_ids.write({
+            'uom_id': self.env.ref('product.product_uom_dozen').id,
+        })
+        self.invoice.invoice_line_ids._onchange_uom_id()
+        self.invoice.invoice_line_ids._onchange_product_id_margin()
```
Comment (chienandalu) [inline account_invoice_margin/tests/test_account_invoice_margin.py]: Travis failing on wrong onchange method call here

File: account_invoice_margin/models/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    )
+    purchase_price = fields.Float(
+        digits=dp.get_precision('Product Price'),
+        string='Cost',
+    )
+
+    @api.multi
+    @api.depends('purchase_price', 'price_subtotal')
+    def _compute_margin(self):
+        for line in self.filtered(lambda x: x.invoice_id.type[:2] != 'in'):
```
Comment (chienandalu) [inline account_invoice_margin/models/account_invoice.py]: This is failing on runbot. Maybe `x.invoice_id and x.invoice_id.type[:2] != 'in'`?

Comment (pedrobaeza) [reply account_invoice_margin/models/account_invoice.py]: Fixed!

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Shouldn't we move this to OCA/margin-analysis?

**Review by chienandalu — APPROVED** [review]
Tested on runbot :+1:

**Comment by pedrobaeza** [comment]
Talked with @sergio-teruel, I'm moving this to OCA/margin-analysis, but merging directly when CIs finish, as it's already approved.

**Comment by pedrobaeza** [comment]
Moved to https://github.com/OCA/margin-analysis/pull/41

**Pull request closed without merging**